zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -225,6 +225,17 @@ function PageWrapper(props) {
   };
 
   const { anchor_el, loading, open_search_form } = state;
+
+  const renderer = useMemo(() => {
```

Comment:
This do not need useMemo. did you see any performance issues?

Reply:
I just wanted to conditionally render some components that depend on certain properties

Reply:
For the scenarios like that we don't need useMemo, only add it if its affecting performance